internal/pkg/template/template.go

```diff
@@ -63,6 +94,34 @@ func (t *Template) Parse(path string, data interface{}, options ...ParseOption)
 	return &Content{buf}, nil
 }
 
+// UploadEnvironmentCustomResources uploads the environment custom resource scripts.
+func (t *Template) UploadEnvironmentCustomResources(upload func(string, ...CustomResource) (string, error)) ([]CustomResource, error) {
+	return t.uploadCustomResources(upload, envCustomResourceFiles)
+}
+
+func (t *Template) uploadCustomResources(upload func(string, ...CustomResource) (string, error), fileNames []string) ([]CustomResource, error) {
+	customResources := []CustomResource{}
+	for _, name := range fileNames {
+		cr := CustomResource{name: envLambdaZippedScriptName}
+		content, err := t.Read(path.Join(envLambdaRootPath, fmt.Sprintf("%s.js", name)))
+		if err != nil {
+			return nil, err
+		}
+		cr.content = content.Bytes()
+		h := sha256.New()
```

Comment:
dumb question: can we just use the [`Sum256`](https://golang.org/pkg/crypto/sha256/#Sum256) fn instead of defining an object?

```go
sum := sha256.Sum256(cr.content)
```

Let's also add a comment explaining _why_ we are doing this, as it probably won't be obvious to us in a few months :D